src/csharp/NativeMethods.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
General comment for the file. Please, add detailed docs for each entry, explaining what parameters are really const, in or out.
Mention C types where appropriate.

It is especially important to understand who owns what and the lifespan of the entities.
For example, it is not clear who owns and destroys OgaGenerate() generated object.

For that we need a clear C API documentation with in/out params and whether the objects and buffers must be destroyed by C# API.

Reply:
I added documentation to both the C API as well as what is in this file.